src/tests/D3MHub.t.sol

```diff
@@ -716,6 +830,18 @@ contract D3MHubTest is DSSTest {
         assertTrue(d3mTestPool.postDebt());
     }
 
+    function test_wind_mcd_caged() public {
+        vat.file("Line", vat.debt() + 45 * RAD);
+        vat.file(ilk, "line", 40 * RAD);
+
+        // MCD shuts down
+        end.cage();
+        end.cage(ilk);
+
+        // try to wind the system
+        _windSystemNoop();
```

Comment:
Is this really a no-op? I think we are unwinding due to mcd being caged.

Reply:
Yeah I think there is a misleading naming here. I think you could also use Foundry's power to check the events, that would allow you to prove better that the winding path is not being taken (what I see is you really trying to test).

Reply:
I also think the internal function makes sense to make a pre state, like in every (or almost) test we need to wind the system to something, then we have `_windSystem()`. However after that I'd expect special conditions and checks happening in the same test itself.